docs/src/tutorials/standalone/Canopy/canopy_tutorial.jl

```diff
@@ -261,59 +264,17 @@ simulation = LandSimulation(
     end_date,
     dt,
     canopy;
-    set_ic! = set_ic!,
-    updateat = updateat,
-    solver_kwargs = (; saveat = deepcopy(saveat)),
+    set_ic!,
+    updateat,
     timestepper = ode_algo,
-    user_callbacks = (saving_cb,),
```

Comment:
you might want to pass an empty set here (user_callbacks = ()) otherwise it will use the NaNchecker and report callback by default.